<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TPlayerWindow</class>
 <widget class="QDialog" name="TPlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generateur aleatoire de joueur</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QSpinBox" name="spinBox_3_hpb">
     <property name="prefix">
      <string>Hp max du boss : </string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_exec">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QSpinBox" name="spinBox_nbr">
     <property name="prefix">
      <string>Nombre de joueurs : </string>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="spinBox_2_deg">
     <property name="prefix">
      <string>DÃ©gats max de chaque joueur : </string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QSpinBox" name="spinBox_xpb">
     <property name="prefix">
      <string>Xp max du boss : </string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
